plugins/wfp/management/commands/south_sudan/Under5.py

```diff
@@ -163,20 +166,20 @@ def run(self):
                 beneficiary = Beneficiary.objects.filter(entity_id=instance["entity_id"]).first()
 
             logger.info("Retrieving journey linked to beneficiary")
-
-            for journey_instance in instance["journey"]:
-                if len(journey_instance["visits"]) > 0:
-                    journey = self.save_journey(beneficiary, journey_instance)
-                    visits = ETL().save_visit(journey_instance["visits"], journey)
-                    logger.info(f"Inserted {len(visits)} Visits")
-                    grouped_steps = ETL().get_admission_steps(journey_instance["steps"])
-                    admission_step = grouped_steps[0]
-
-                    followUpVisits = ETL().group_followup_steps(grouped_steps, admission_step)
-                    steps = ETL().save_steps(visits, followUpVisits)
-                    logger.info(f"Inserted {len(steps)} Steps")
-                else:
-                    logger.info("No new journey")
-            logger.info(
-                f"---------------------------------------------------------------------------------------------\n\n"
-            )
+            if beneficiary is not None:
```

Comment:
Should there be some error notification or something if `beneficiary` is `None`?

Reply:
yep, on QA, there a some strange data with beneficiary with medical visit without anthropometric during admission visit.It has happened for Children under 5 only for South Sudan and it was violating some DB constraint.